pkg/wireguard/agent/agent.go

```diff
@@ -454,6 +561,38 @@ func (a *Agent) DeletePeer(nodeName string) error {
 	return nil
 }
 
+func (a *Agent) reassemblyMasterHubConfig( ) error {
+      // first clean up peerMasternode's allowIps
+      filtered := a.peerMasternode.allowedIPs[:0]
```

Comment:
`filtered` is misleading?

```suggestion
      resetSlice := a.peerMasternode.allowedIPs[:0]
```

Reply:
will adjust the code